llama_index/__init__.py

```diff
@@ -151,28 +88,11 @@
     # indices
     "SummaryIndex",
     "VectorStoreIndex",
-    "SimpleKeywordTableIndex",
```

Comment:
I think moving the keyword, knowledge graph, and document summary index imports will cause some issues.

Reply:
my thought was to keep the most common ones covering >90 what people are using. 

good pushback, I'll add those back.